schema/deploy/shipping/views.sql

```diff
@@ -1029,6 +1029,12 @@ grant select
    to "incidence-modeler";
 
 
+revoke all
+    on shipping.hcov19_observation_v1
+  from "hcov19-visibility";
```

Comment:
Is this step necessary before dropping a view just to replace it?

Reply:
(It isn't.)

Reply:
Huh, not sure how I got it in my head that it was necessary...will remove